src/config/babel.js

```diff
@@ -9,7 +9,10 @@ export default async function ({projectRoot, preset, transpileLint}) {
   }
 
   if (preset) {
-    await fsPromises.writeFile(`${projectRoot}/.babelrc`, JSON.stringify({presets: [preset.name], ignore: ['./lib/']}));
+    await fsPromises.writeFile(
+      `${projectRoot}/.babelrc`,
+      JSON.stringify({presets: [preset.name], ignore: ['./lib/'], env: {test: {plugins: ['istanbul']}}})
```

Comment:
does some sort of dependency need to be installed to support the `istanbul` plugin here?